crates/ruff_python_formatter/src/comments/mod.rs

```diff
@@ -178,7 +178,14 @@ impl SourceComment {
                 "off" => Some(SuppressionKind::Off),
                 "on" => Some(SuppressionKind::On),
                 "skip" => Some(SuppressionKind::Skip),
-                _ => None,
+
+                // Allow additional context separated by whitespace
+                command => match command.split_whitespace().next() {
```

Comment:
Could we just replace `match command.trim_whitespace_start()` with `match command.trim_whitespace_start(). split_whitespace().next()`? That is, could we just always split by whitespace and take the first token, rather than nesting the match expressions?

Reply:
Yes that's how I implemented it at first but then I was wondering if we should only allow comments on `fmt: off` and `fmt: skip` but not `fmt: on` which led me to this structure. I decided it'd be weird not to support `fmt: on - reason` though and I can see if being useful. I left it this way because I figured there may be a performance benefit and it's still easy to read. I'm happy to go back though.